apps/google-analytics-4/lambda/src/app.spec.ts

```diff
@@ -8,12 +8,20 @@ import {
 } from '../test/mocks/googleApi';
 import app from './app';
 import { GoogleApi } from './services/google-api';
+import * as NodeAppsToolkit from '@contentful/node-apps-toolkit';
 
 chai.use(chaiHttp);
 
 describe('app', () => {
   let mockClient: SinonStubbedInstance<AnalyticsAdminServiceClient>;
 
+  beforeEach(() => {
+    // TODO: set headers and fully test signature verification later
```

Comment:
Still valid TODO? Let's also make sure to pull into subtask and not make it a someday.

Reply:
👍